.github/workflows/nightly-tests.yml

```diff
@@ -50,8 +50,10 @@ jobs:
       run: rustup component add rust-src --toolchain nightly-2023-01-03-x86_64-unknown-linux-gnu
     - name: Install pilcom
       run: git clone https://github.com/0xPolygonHermez/pilcom.git  && cd pilcom && npm install
+    - name: Check without Halo2
+      run: cargo check
```

Comment:
Why not check with halo2?

Reply:
Because it is built with halo2 later, and build includes check.